translations/pt.yaml

```
@@ -663,7 +663,7 @@ model:
       title: ""
     ElPlanetaDeNano:
       title: ""
-    Dibujando Figuras:
+    Desenho Figuras:
```

```suggestion
    Dibujando Figuras:
```

Aca quedo traducido un tag, habria que cambiarlo de vuelta a español para que pueda buildear pilas bloques.